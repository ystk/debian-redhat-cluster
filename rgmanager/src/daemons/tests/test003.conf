<?xml version="1.0"?>
<!-- Add an initscript to our regular resource group.
     The initscript should get a 'NEEDSTART' flag. -->
<cluster>
<rm>
	<resources>
    		<service name="test1"/>
		<script name="initscript" file="/etc/init.d/httpd"/>
	</resources>
	<service ref="test1">
		<script ref="initscript"/>
	</service>
</rm>
</cluster>
