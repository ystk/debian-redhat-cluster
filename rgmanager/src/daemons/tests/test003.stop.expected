Stopping test1...
[stop] script:initscript
[stop] service:test1
Stop of test1 complete
